src/tests/test_integration/test_shopping_cart.py

```
@@ -3,12 +3,13 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 from database.models.shopping_cart import Cart, CartItem
-from database.models.movies import MovieModel, CountryModel, MovieStatusEnum
+from database.models.movies import MovieModel
 from database.models.accounts import UserModel, UserGroupModel, UserGroupEnum
 from database.models.base import Base
 from httpx import AsyncClient
 from sqlalchemy.ext.asyncio import AsyncSession
 from tests.conftest import client, db_session, seed_user_groups
```

Ruff (0.11.9)

`tests.conftest.client` imported but unused

Remove unused import

(F401)